src/collective/solr/timeout.py

```diff
@@ -1,39 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
@tisto It's included in Python 2.6+ see https://github.com/collective/collective.solr/commit/6334d9e007bc576691659e7c22199015b634ea7f